picard/formats/apev2.py

```diff
@@ -136,6 +137,8 @@ def _load(self, filename):
         file = self._File(encode_filename(filename))
         metadata = Metadata()
         if file.tags:
+            config = get_config()
+            date_sanitize = not config.setting['disable_date_sanitize'] and not any('.ape' in fmt[0] for fmt in config.setting['formats_to_disable_date_sanitize'])
```

Comment:
I don't think we still need `not config.setting['disable_date_sanitize']`.

Also I think it should apply to **file format** not **file extensions** (one format, multiple extensions).

The test should be  as simple as:

```python
date_sanitize = self.NAME in config.setting['formats_to_disable_date_sanitize'])
```

See https://github.com/metabrainz/picard/blob/226ebef845151cdb3a89698e5802340b29edfc57/picard/formats/apev2.py#L354-L373 , basically all formats have a unique name (but each can match multiple file extensions)